login.html

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FinestStitch</title>
+    <link rel="stylesheet" href="https://distracted-jones-2f14b5.netlify.app/index.css" />
```

Comment:
You imported this library in style.css too, you may not need to import it here :)

Reply:
okay..removing it